build/includes/terraform.mk

```
@@ -15,11 +15,11 @@
 ### Deploy cluster with Terraform
 terraform-init:
```

Just noticed that we forgot to add rebuild of main build image.
```suggestion
  terraform-init:
  terraform-init: $(ensure-build-image)
```